generate_openj9_scc.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+# Format taken from OpenLiberty SCC generation in docker images
+
+unset OPENJ9_JAVA_OPTIONS
+
+SCC_SIZE="50m" # Default size for SCC
+SCC_GEN_RUNS_COUNT=3 # Runs for generating SCC
+
+run_eclipse_and_stop() {
+
+    "$HOME"/eclipse-home/eclipse/eclipse &
+    ECLIPSE_PID=$!
+    
+    sleep 1m
+
+    kill -9 $ECLIPSE_PID
+    
+    sleep 10s
+
+}
+
+download_eclipse() {
+
+    # Creating a temporary directory for tomcat download
+    mkdir -p "$HOME"/eclipse-home/
+
+    # Downloading eclipse
+    wget --directory-prefix=/tmp  http://www.mirrorservice.org/sites/download.eclipse.org/eclipseMirror/technology/epp/downloads/release/2020-06/M3/eclipse-java-2020-06-M3-linux-gtk-x86_64.tar.gz
+
+    # Extracting eclipse
+    tar -xvzf /tmp/eclipse-java-2020-06-M3-linux-gtk-x86_64.tar.gz -C "$HOME"/eclipse-home
+
+    # Removing the tar file
+    rm -rf /tmp/eclipse-java-2020-06-M3-linux-gtk-x86_64.tar.gz
+
+}
+
+install_packages() {
+    apt update
+    DEBIAN_FRONTEND=noninteractive apt install -y gtk+3.0 xvfb
+}
+
+dry_run_eclipse() {
+
+    # Creating base layer first instead of running tomcat server to generate SCC directly
+    java -Xshareclasses:name=default_scc,cacheDir=/opt/java/.scc,bootClassesOnly,nonFatal,createLayer -Xscmx$SCC_SIZE -version
+
+    # Pointing cache for tomcat runs
+    export OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=default_scc,cacheDir=/opt/java/.scc,bootClassesOnly,nonFatal"
+
+    Xvfb :1 -ac -screen 0 1024x768x8 &
```

Comment:
Please add comments so we can understand why you are doing this

Reply:
Sorry for missing it earlier, added now